models/news.js

```diff
@@ -55,4 +55,23 @@ exports.selectCommentsByArticleId = (articleId) => {
     .then((comments) => {
         return comments.rows
     })
+}
+
+exports.insertComment = (articleId,newComment) => {
+    if(!Number(articleId)){
+        return Promise.reject({status:400, msg:'id is not a number'})
+    }
```

Comment:
Be Careful as this could evaluate to 0 which is a falsy value. Its better to let sql handle this or do some extra checks